### PR TITLE
Updates to the README

### PR DESCRIPTION
**Pull request by kehoecj**
This PR contains the following changes:

1. Updates to the README ([view processed README](https://github.com/Boeing/config-file-validator/blob/c207401a0e28ae4d9ec14f115b7796f290f33eb9/README.md)) including a new logo, new badges, and new sections for downloading the executables from Github releases or using `go install`
2. Adds coveralls to the pipeline for code coverage (plus badge)
3. Adds documentation to the validator command which will appear in the pkg.go.dev docs instead of the current [blank page](https://pkg.go.dev/github.com/Boeing/config-file-validator/cmd/validator)

Please feel free to give frank opinions on logo, design, etc

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,7 +1,32 @@
...
+  <a href="https://goreportcard.com/report/github.com/Boeing/config-file-validator">
+  <img src="https://goreportcard.com/badge/github.com/Boeing/config-file-validator" alt="Go Report Card">
+  </a>
+
+  <a href="https://github.com/boeing/config-file-validator/actions/workflows/go.yml">
+  <img src="https://github.com/boeing/config-file-validator/actions/workflows/go.yml/badge.svg" alt="Pipeline Status">
+  </a>
+  
+  <a href='https://coveralls.io/github/kehoecj/config-file-validator?branch=readme_updates'>
+    <img src='https://coveralls.io/repos/github/kehoecj/config-file-validator/badge.svg?branch=readme_updates' alt='Coverage Status' />
```
Comment (jd4235) [inline README.md:24]: This is tied to your repo and specific branch. Assuming this should be updated to point to the boeing repo and mainline branch.

**Review by jd4235 — CHANGES_REQUESTED** [review]
See comment.

File: README.md (hunk 1 above)
Comment (kehoecj) [reply README.md:24]: Put in a request to the Boeing org to allow access to coveralls - if rejected I may need to leave the badge out for now and pursue a different path 🤞

Comment (kehoecj) [reply README.md]: Looks like approval will take a while. I removed the coverage badge and job and will re-add after approval

**Pull request merged**